templates/16-04/index.html

```diff
@@ -11,7 +11,7 @@
       <h1>Ubuntu 16.04 LTS <br class="u-hide--small">(Xenial Xerus)</h1>
       <p>Ubuntu 16.04 LTS (Xenial Xerus) was released on 21 April 2016 and introduced us to snaps, a more secure app format, with faster and simpler updates, as well as LXD, the pure-container hypervisor and OpenStack Mitaka. </p>
       <h2 class="p-heading--4">Is Ubuntu 16.04 LTS still supported?</h2>
-      <p class="u-sv2">Yes, Ubuntu 16.04 LTS is supported until 2024 through Canonical’s Extended Security Maintenance (ESM) product. Xenial enters the ESM period in April 2021, with security patches provided for an additional three years beyond the traditional five-year standard support.</p>
+      <p class="u-sv2">Yes, Ubuntu 16.04 LTS is supported until 2024 through Canonical’s Extended Security Maintenance (ESM) product. Xenial entered the ESM period in April 2021, with security patches provided for an additional three years beyond the traditional five-year standard support.</p>
```

Comment:
```suggestion
      <p class="u-sv2">Yes, Ubuntu 16.04 LTS is supported until 2026 through Canonical’s Extended Security Maintenance (ESM) product. Xenial entered the ESM period in April 2021, with security patches provided for an additional three years beyond the traditional five-year standard support.</p>
```

Reply:
If I understand the comment correctly we need to wait until September 15th to make that change 
![image](https://user-images.githubusercontent.com/11927929/129006725-72a97736-d504-415e-b1a0-f4a1d4448213.png)